app/policies/case/sar/offender_policy.rb

```diff
@@ -45,6 +45,11 @@ def can_record_data_request?
     check_can_trigger_event(:add_data_received)
   end
 
+  def can_send_day_1_email
```

Comment:
All the other methods use a `?`, and you are using the question mark when you call the method.